### ISSUE TITLE
Rendered where clauses in `fn` docs need more linebreaks

### ISSUE THREAD

**Issue by huonw** [rust-lang/rust#24373]
E.g. http://doc.rust-lang.org/nightly/std/thread/fn.scoped.html is all on one line

```rust
pub fn scoped<'a, T, F>(f: F) -> JoinGuard<'a, T> where T: Send + 'a, F: FnOnce() -> T, F: Send + 'a
```

It'd be much nicer as

```rust
pub fn scoped<'a, T, F>(f: F) -> JoinGuard<'a, T>
    where T: Send + 'a, F: FnOnce() -> T, F: Send + 'a
```

or even broken more.

**Comment by jooert** [comment]
This was fixed in #25933, I guess.

**Comment by steveklabnik** [comment]
Looks so!
